pkg/kcov/cdefs.go

```diff
@@ -0,0 +1,43 @@
+package kcov
+
+import "unsafe"
+
+const (
+	// sizeof(uintptr)
+	sizeofUintPtr = 8
+)
+
+const (
+	_IOC_NRBITS   = 8
+	_IOC_TYPEBITS = 8
+	_IOC_SIZEBITS = 14
+	_IOC_DIRBITS  = 2
+
+	_IOC_NRSHIFT   = 0
+	_IOC_TYPESHIFT = _IOC_NRSHIFT + _IOC_NRBITS
+	_IOC_SIZESHIFT = _IOC_TYPESHIFT + _IOC_TYPEBITS
+	_IOC_DIRSHIFT  = _IOC_SIZESHIFT + _IOC_SIZEBITS
+
+	_IOC_NONE  = 0
+	_IOC_WRITE = 1
+	_IOC_READ  = 2
+)
+
+// KCOV ioctl commands for Linux.
+const (
```

Comment:
Will it be more readable to declare something like 
```
func ioctl(dir, t, nr, size uintptr) uintptr {
	return (dir << _IOC_DIRSHIFT) | (size << _IOC_SIZESHIFT) | (t << _IOC_TYPESHIFT) | (nr << _IOC_NRSHIFT)
}
```
and then use it to declare the consts?

Reply:
Yeah, my bad.
I think it's good as is then.